src/config.rs

```diff
@@ -132,6 +140,9 @@ impl Config {
             );
             self.recovery_threads = MIN_RECOVERY_THREADS;
         }
+        if !Version::is_valid(self.format_version) {
+            self.format_version = Version::default() as u64;
```

Comment:
Warning before changing the config value.

Reply:
Done.